src/components/Capture/index.js

```diff
@@ -11,4 +11,4 @@ export const FrontDocumentCapture = appendToTracking(withOptions(Document), 'fro
 export const BackDocumentCapture = appendToTracking(withOptions(Document, { side: 'back' }), 'back_capture')
 export const SelfieCapture = appendToTracking(withOptions(Face, { requestedVariant: 'standard' }), 'selfie_capture')
 export const VideoCapture = appendToTracking(withOptions(Face, { requestedVariant: 'video' }), 'video_capture')
-export const PoADocumentCapture = appendToTracking(Document, 'poa')
+export const PoADocumentCapture = appendToTracking(withOptions(Document, { forceCrossDevice: false }), 'poa')
```

Comment:
do we really need to make it not available for poa?

Reply:
It's part of the acceptance criteria

Reply:
Tagging @seewah  for more details/offline conversation.

Reply:
yes the primary option for PoA is upload (e-statement, etc) this is a big product assumption we have validated. So, i say, yes the option is document step only, if that is what we are talking about